content/en/docs/reference/config-api/apiserver-config.v1.md

```diff
@@ -38,6 +39,96 @@ auto_generated: true
 </tbody>
 </table>
 
+## `EncryptionConfiguration`     {#apiserver-config-k8s-io-v1-EncryptionConfiguration}
+    
+
+
+<p>EncryptionConfiguration stores the complete configuration for encryption providers.
+It also allows the use of wildcards to specify the resources that should be encrypted.
+Use '&ast;.&lt;group&gt;' to encrypt all resources within a group or '&ast;.&ast;' to encrypt all resources.
+'&ast;.' can be used to encrypt all resource in the core group.  '&ast;.&ast;' will encrypt all
+resources, even custom resources that are added after API server start.
+Use of wildcards that overlap within the same resource list or across multiple
+entries are not allowed since part of the configuration would be ineffective.
```

Comment:
```suggestion
entries are not allowed, since part of the configuration would be ineffective.
```

Reply:
since this is a result of the reference generator, this change would have to be made at the source